src/client/modules/my/donutChart/donutChart.js

```diff
@@ -0,0 +1,46 @@
+import { api, LightningElement } from 'lwc';
+import { DonutChart } from 'my/d3Charts';
+
+export default class ConnectedScatterGraph extends LightningElement {
+
+    @api graphTitle = "Place Frequency Graph";
+
+    // Must be in the form [{name: string, value: number}]
```

Comment:
DonutChart is notably simpler and less configurable, so it's fairly straight forward to re-work into this form, and have the component just plop it into D3